arangod/Replication2/ReplicatedLog/types.h

```diff
@@ -236,4 +236,30 @@ struct QuorumData {
   void toVelocyPack(velocypack::Builder& builder) const;
 };
 
+namespace {
+constexpr std::string_view kStringUnconfigured = "Unconfigured";
+constexpr std::string_view kStringRecovery = "RecoveryInProgress";
+constexpr std::string_view kStringOperational = "ServiceOperational";
+}  // namespace
+
+enum class LocalStateMachineStatus { kUnconfigured, kRecovery, kOperational };
+auto to_string(LocalStateMachineStatus) noexcept -> std::string_view;
+
+struct LocalStateMachineStatusStringTransformer {
```

Comment:
Since this transformer is not implemented anywhere, I think it's best to not have it here. I could see myself trying to use it in the future and reporting that there's a problem.

Reply:
Gone!